lib/galaxy/webapps/galaxy/api/trs_consumer.py

```diff
@@ -21,12 +21,12 @@ def get_servers(self, trans, *args, **kwd):
 
```

Comment:
Just noticed that this (`/api/trs_consume/`) and `/api/trs_search` don't have any API test in `lib/galaxy_test/api/` . Not the fault of this PR though, and I guess the endpoints are exercised by the Selenium tests.